requirements/requirements.txt

```diff
@@ -1,5 +1,7 @@
 # model
-openai[embeddings]~=0.27
+openai~=1.37
+transformers[sentencepiece]~=4.42
+jinja2>=3.1.0
```

Comment:
please freeze particular version so future releases of Jinja won't break it

Reply:
Can be changed to `jinja2~=3.1.0`